docs/content/concepts/_index.md

```diff
@@ -1,7 +1,109 @@
 ---
 type: docs
-title: "Radius concepts"
+title: "Radius Concepts"
 linkTitle: "Concepts"
-description: "Learn about Radius background, core concepts, and architecture"
-weight: 80
+description: "Radius core concepts and architecture"
+weight: 20
 ---
+
+## Introduction
+
+Radius is a platform for defining and deploying applications and infrastructure to any cloud. It is a central component of modern-day internal developer platforms (IDPs). It allows platform engineers to define resource types for developers to use when building their applications, and separately, the implementation of those resource types using existing Infrastructure as Code (IaC) templates and modules. Additionally, Radius enables platform engineers to define logical environments with specific deployment targets (e.g., a specific cloud provider region), each with their own IaC implementation.
+
+This page provides a conceptual overview of Radius. It describes Radius' logical components, technical architecture and how Radius relates to other IDP components. It is accompanied by additional concept pages focused on each of the core components: Resource Types, Recipes, Environments, and Applications. If you are new to Radius, you are encouraged to complete the [Quick Start]({{< ref "quick-start" >}}). Then, after reading the concept documentation, complete the end-to-end [tutorial]({{< ref "tutorials" >}}).
+
+## IDP reference architecture
```

Comment:
I think it might be useful to frame this reference architecture in the context of one that is centered around Radius and how it helps materialize a more ideal architecture